### PR TITLE
Bugfix/update sponsors

### LINKED ISSUE #379: Update views
Update _Sponsors_ view with "w tamtym roku wspierali nas" and uncomment section _promot_section_
Update _Prizes_ view with "w tamtym roku otrzymaliście nagrody" and uncomment div with information about sponsoring

### PR DESCRIPTION
**Pull request by szrtGreg**
## Description
- Changes in view (Prizes, Sponsors)
- fix 'Wyniki" link

## Where should the reviewer start?
Files affected: views, resource folder (in main project) and _Layout
## Motivation and context
closes #379 

## How has this been tested?
Sponsors:
obraz

Prizes:
obraz

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by WTobor** [comment]
@szrtGreg please remember to use resources ;)

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx [hunk 1]
```diff
@@ -243,7 +243,10 @@
   <data name="DontMissEvents" xml:space="preserve">
     <value>I nie przegap najważniejszych wydarzeń!</value>
   </data>
-  <data name="DoYouWantBeSponsor2019" xml:space="preserve">
+  <data name="DoYouWantBeSponsor2020" xml:space="preserve">
+    <value>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2020 roku?</value>
+  </data>
+  <data name="DoYouWantToBeASponsor" xml:space="preserve">
     <value>Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku?</value>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx:250]: please remove that part, we don't need it anymore (it is no longer 2020 year)

Comment (szrtGreg) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx:250]: Do you mean to remove "Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2020 roku?" ?

Comment (szrtGreg) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsMessages.resx]: OK, I got your point. I removed not used  resource  DoYouWantBeSponsor - Chcesz zostać sponsorem i wesprzeć dobrą inicjatywę w 2019 roku) but still we need DoYouWantBeSponsor2020 which i left

**Review by WTobor — APPROVED** [review]

**Pull request merged**